tests/unit/docs/test_client.py

```diff
@@ -144,18 +148,22 @@ def test_modeled_exceptions(self):
         self.assert_contains_lines_in_order(
             [
                 '.. py:class:: MyService.Client.exceptions.SomeException',
-                '**Example**::',
+                '  .. raw:: html',
+                '    <h3>Example</h3>',
+                '::',
                 'except client.exceptions.SomeException as e:',
                 '.. py:attribute:: response',
-                '**Syntax**',
+                '    .. raw:: html',
+                '      <h3>Syntax</h3>',
                 '{',
                 "'Message': 'string',",
                 "'Error': {",
                 "'Code': 'string',",
                 "'Message': 'string'",
                 '}',
                 '}',
-                '**Structure**',
+                '    .. raw:: html',
+                '      <h3>Structure</h3>',
```

Comment:
* Additional indentation - This is just me applying stricter validation that not only validates content but its indentation as well. This assertion checks if a provided string is a substring of a pages content so if the true value of a line in our docs was `    ..rst:: html` and we made an unintentional update that changed it to ` ..rst:: html` it would continue to pass if we were validating with ` ..rst:: html` but fail with `    ..rst:: html` since it expects more indentation. This is good for catching future unintended whitespace issues but can also be a pain if we make an intentional whitespace change.

* Indentation Variance - Some lines may have more indentation than others since they're more nested than other headings. If you look at this [example page](https://botocore.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/exceptions/NoSuchBucket.html), you'll see that `Syntax` and `Structure` are more nested than `Examples`.